Project-Aurora/Project-Aurora/Devices/Corsair/CorsairDevice.cs

```diff
@@ -381,41 +372,62 @@ private void SendColorToPeripheral(Color color, bool forced = false)
+                    if (commanderPro != null && Global.Configuration.VarRegistry.GetVariable<bool>($"{devicename}_corsair_commander"))
+                    {
+                        commanderLED.Brush = (CUE.NET.Brushes.SolidColorBrush)color;
+                        commanderPro.Update(true);
+                    }
+                    if (lightingNode != null && Global.Configuration.VarRegistry.GetVariable<bool>($"{devicename}_corsair_lighting"))
                     {
-                        if (mousemat[CorsairLedId.Zone1] != null)
-                            mousemat[CorsairLedId.Zone1].Color = color;
-                        if (mousemat[CorsairLedId.Zone2] != null)
-                            mousemat[CorsairLedId.Zone2].Color = color;
-                        if (mousemat[CorsairLedId.Zone3] != null)
-                            mousemat[CorsairLedId.Zone3].Color = color;
-                        if (mousemat[CorsairLedId.Zone4] != null)
-                            mousemat[CorsairLedId.Zone4].Color = color;
-                        if (mousemat[CorsairLedId.Zone5] != null)
-                            mousemat[CorsairLedId.Zone5].Color = color;
-                        if (mousemat[CorsairLedId.Zone6] != null)
-                            mousemat[CorsairLedId.Zone6].Color = color;
-                        if (mousemat[CorsairLedId.Zone7] != null)
-                            mousemat[CorsairLedId.Zone7].Color = color;
-                        if (mousemat[CorsairLedId.Zone8] != null)
-                            mousemat[CorsairLedId.Zone8].Color = color;
-                        if (mousemat[CorsairLedId.Zone9] != null)
-                            mousemat[CorsairLedId.Zone9].Color = color;
-                        if (mousemat[CorsairLedId.Zone10] != null)
-                            mousemat[CorsairLedId.Zone10].Color = color;
-                        if (mousemat[CorsairLedId.Zone11] != null)
-                            mousemat[CorsairLedId.Zone11].Color = color;
-                        if (mousemat[CorsairLedId.Zone12] != null)
-                            mousemat[CorsairLedId.Zone12].Color = color;
-                        if (mousemat[CorsairLedId.Zone13] != null)
-                            mousemat[CorsairLedId.Zone13].Color = color;
-                        if (mousemat[CorsairLedId.Zone14] != null)
-                            mousemat[CorsairLedId.Zone14].Color = color;
-                        if (mousemat[CorsairLedId.Zone15] != null)
-                            mousemat[CorsairLedId.Zone15].Color = color;
-
-                        mousemat.Update(true);
-                    }*/
+                        lightingLED.Brush = (CUE.NET.Brushes.SolidColorBrush)color;
+                        lightingNode.Update(true);
+                    }
+                    if (memory != null && Global.Configuration.VarRegistry.GetVariable<bool>($"{devicename}_corsair_memory"))
```

Comment:
`{devicename}_corsair_memory` should be `{devicename}_corsair_ram`, as set in the registry (L872). It's always returning false currently so memory modules don't light up.

Reply:
Thanks. Check #1530